src/UI/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -7,16 +7,25 @@
     <PackageTags>aspnetcore;identity;membership;razorpages</PackageTags>
     <EnableApiCheck>false</EnableApiCheck>
     <GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
+    <RazorCompileOnBuild>true</RazorCompileOnBuild>
```

Comment:
Add `<ResolvedRazorCompileToolset>RazorSdk</ResolvedRazorCompileToolset>`

This is a feature that the Web SDK has added for us, but it hasn't made it into our builds yet.